app/views/claims/_school_search.html.erb

```diff
@@ -0,0 +1,19 @@
+<% if params[:school_search].present? && current_claim.errors.empty? %>
+  <p class="govuk-body">
+    <strong>No results match that search term. Try again.</strong>
+  </p>
+<% else %>
+  <span id="school-search-hint" class="govuk-hint">
+    Search for the school name, use at least four characters.
+  </span>
+<% end %>
+
+<%= form_tag(path, method: :get) do %>
+  <div class="govuk-form-group">
+    <%= label_tag :school_search, "School name", class: "govuk-label" %>
+    <%= text_field_tag :school_search, params[:school_search], class: "govuk-input", value: school&.name %>
+  </div>
+  <div class="govuk-form-group">
+    <%= submit_tag "Search", class: "govuk-button" %>
+  </div>
+<% end %>
```

Comment:
Did anything change when refactoring these? If not, would be a massive pain to roll the refactor back? The changes to the autocomplete involve a different refactor that will be a pain to merge with this.

Reply:
Not a huge change. Happy to roll it back 👍